Code/Framework/AzFramework/AzFramework/PaintBrush/PaintBrushSystemComponent.cpp

```diff
@@ -51,7 +51,7 @@ namespace AzFramework
 
     AZ::Uuid PaintBrushSystemComponent::StartPaintSession(const AZ::EntityId& entityId)
     {
-        AZ::EntityComponentIdPair entityComponentPair;
+        AZ::EntityComponentIdPair entityComponentPair(AZ::EntityId(AZ::EntityId::InvalidEntityId), AZ::InvalidComponentId);
```

Comment:
There are many places down in AZCore that follow this pattern, I believe it's to avoid the cost of construction in cases where the values are about to get assigned to something. I'm not personally a fan of the pattern, since it's a bit too easy to get bitten by uninitialized variables (AABB I'm looking at you), but seemingly small construction costs can add up and cause death by 1000 cuts, so I get it. It's a choice of which pain to accept. :)